vNext/test/ServiceBus.Tests/ServiceBus.Tests.csproj

```diff
@@ -23,7 +23,7 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="FluentAssertions" Version="4.0.0" />
+    <PackageReference Include="FluentAssertions" Version="4.18.0" />
```

Comment:
I'm not sure, but I saw relevant errors in vNext tests. I made this change & it fixed them